test_programs/compile_success_empty/comptime_as_slice/src/main.nr

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    comptime {
```

Comment:
```suggestion
    comptime
    {
```
The formatter is a bit weird

Reply:
Yeah, I'm not a fan of this either https://github.com/noir-lang/noir/issues/5281